packages/core/src/commands/config-cli.ts

```diff
@@ -63,9 +63,10 @@ export class Command extends Commands.Command {
      * @memberof Command
      */
     public async execute(): Promise<void> {
-        if (this.hasFlag("token")) {
-            this.config.set("token", this.getFlag("token"));
-        }
+        // TODO: check
```

Comment:
That must've been a copy & paste mistake on my end when I created the command. Since this is a command to alter the configuration nothing should have a default value which means `token` could actually be empty.